src/com/pentalog/pentastagiu/homework2/ConsoleApp/Candy.java

```diff
@@ -0,0 +1,21 @@
+package com.pentalog.pentastagiu.homework2.ConsoleApp;
+
+import javax.naming.Name;
+
+public class Candy extends Product {
+    private int Quantity;
+
+
+    Candy(int Price,String Name,int Quantity){
+        super(Price,Name);
+        this.Quantity=Quantity;
+    }
+
+    public int getQuantity() {
+        return Quantity;
+    }
+
+    public void setQuantity(int Quantity) {
```

Comment:
si aici, daca alegi sa pastrezi setterul, trebuie sa te asiguri ca nu lasi setarea unei cantitati negative;
dar cel mai bine e sa incepi implementarea cu fielduri private si sa adaugi getteri/setteri pe masura ce ai nevoie de ei in implementare si daca chiar ai nevoie sa te asiguri ca ai adaugat validarile necesare pentru a impiedica crearea unor obiecte corupte